jobdata.json

```diff
@@ -0,0 +1,12 @@
+[
```

Comment:
Assuming the placement of this json file is not intended?

Reply:
good catch